WalletLibrary/WalletLibrary/VerifiedIdClientBuilder.swift

```diff
@@ -24,8 +24,9 @@ public class VerifiedIdClientBuilder {
 
     /// Builds the VerifiedIdClient with the set configuration from the builder.
     public func build() -> VerifiedIdClient {
-        /// TODO: inject log consumer and access group identifier into vc sdk.
-        let _ = VerifiableCredentialSDK.initialize()
+        /// TODO: access group identifier into vc sdk.
```

Comment:
what happens if access group identifier is not sent?

Reply:
its okay if it is nil